App.tsx

```diff
@@ -2,18 +2,19 @@ import React, { Component } from 'react'
 import { StyleSheet, View } from 'react-native'
 import { Provider } from 'react-redux'
 import { store } from './src/store'
-import Wrapper from "./views/Wrapper"
+import Wrapper from './views/Wrapper'
+import SocketProvider from './components/SocketProvider'
 
 export default class App extends Component {
```

Comment:
This could be made into a functional component, since we don't use any state or lifecycle methods